test/browser-tests/twoway.js

```diff
@@ -603,7 +603,11 @@ test( 'Changes made in oninit are reflected on render (#1390)', t => {
 	new Ractive({
 		el: fixture,
 		template: '{{#each items}}<input type="checkbox" name="{{selected}}" value="{{this}}">{{/each}}',
-		data: { items: [ 'a', 'b', 'c' ] },
+		data: {
+			items: [ 'a', 'b', 'c' ],
+			// will fail without this
+			selected: null
+		},
```

Comment:
LOL! I see what you're saying there: yak shaving makes more sense for a PR relevant to that specific error :)